lib/rust/ensogl/example/slider/src/lib.rs

```diff
@@ -24,20 +24,286 @@
 use ensogl_core::prelude::*;
 use wasm_bindgen::prelude::*;
 
+use enso_frp as frp;
+use ensogl_core::application::shortcut;
 use ensogl_core::application::Application;
+use ensogl_core::application::View;
 use ensogl_core::data::color;
-use ensogl_core::display::object::ObjectOps;
+use ensogl_core::display;
 use ensogl_hardcoded_theme as theme;
 use ensogl_slider as slider;
 use ensogl_text_msdf::run_once_initialized;
 
 
 
+// ===================================
+// === Basic slider initialization ===
+// ===================================
+
+/// Create a basic slider.
+fn make_slider(app: &Application) -> slider::Slider {
+    let slider = app.new_view::<slider::Slider>();
+    slider.frp.set_background_color(color::Lcha(0.8, 0.0, 0.0, 1.0));
+    slider.frp.set_max_value(5.0);
+    slider.frp.set_default_value(1.0);
+    slider.frp.set_value(1.0);
```

Comment:
I understand that even if these things are not set, the slider would be displayed in a "default mode", right?

Reply:
Yes. This is just to have a larger range and a non-zero default for the example sliders, and a non-gray track color to visualize the graying out of the disabled slider.